<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>33</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(63, 177, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnVender">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>VENDER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnProductos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>PRODUCTOS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClientes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>CLIENTES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAlmacenes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>ALMACENES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnProveedores">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>PROVEEDORES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSalidas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>SALIDAS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVentas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>VENTAS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDetalleVentas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>161</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

background-color: rgb(89, 103, 255);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>DETALLE VENTAS</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>0</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 109, 101);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSalir">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

font:14pt &quot;Monaco&quot;;

}

QPushButton:hover{

	background-color: rgb(255, 0, 12);
font:14pt &quot;Monaco&quot;;

}</string>
     </property>
     <property name="text">
      <string>SALIR</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.451628, y1:0.915, x2:0.493611, y2:0.0454545, stop:0.142857 rgba(210, 31, 255, 255), stop:1 rgba(5, 161, 253, 255));
font: 18pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/logo_gicson.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>-37</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
